### PR TITLE
[Feature]: Added auth-pages

### PR DESCRIPTION
**Pull request by parthpandyappp**
Added _**auth**_  pages with `login.html` & `signup.html`
Below are the deployed PR screens

Direct Link to Signup page: https://deploy-preview-1--penandpap.netlify.app/pages/markup/signup.html
Direct Link to Login page: https://deploy-preview-1--penandpap.netlify.app/pages/markup/login.html

### CODE REVIEW

File: pages/styles/login.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  margin-top: 5rem;
+  height: 100%;
+}
+
+.auth-form form {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  /* background-color: var(--color-primary); */
+  /* color: var(--color-basic); */
```
Comment (NeerajVodala) [inline pages/styles/login.css]: Delete the commented code before pushing it to the dev

**Review by NeerajVodala — APPROVED** [review]
Looks good to me (left one comment)
All the best.

File: pages/styles/login.css (hunk 1 above)
Comment (parthpandyappp) [reply pages/styles/login.css]: Done :+1:

Comment (parthpandyappp) [reply pages/styles/login.css]: Done :+1:

File: pages/styles/login.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+.login-btn {
+  padding: 0.5rem !important;
+}
+
+.text-center {
+  text-align: center;
+}
+
+input {
```
Comment (rishbitsnbytes) [inline pages/styles/login.css]: Better to use dedicated class names for 2 reasons :- 1. Consistency. 2. To avoid conflicting as project gets bigger.

**Review by rishbitsnbytes — COMMENTED** [review]
Added one comment, rest looks good to me.

File: pages/styles/login.css (hunk 2 above)
Comment (parthpandyappp) [reply pages/styles/login.css]: Thanks, I've fixed it.

**Pull request merged**